mmdet/core/evaluation/bbox_overlaps.py

```diff
@@ -1,21 +1,32 @@
 import numpy as np
 
 
-def bbox_overlaps(bboxes1, bboxes2, mode='iou', eps=1e-6):
+def bbox_overlaps(bboxes1,
+                  bboxes2,
+                  mode='iou',
+                  eps=1e-6,
+                  use_legacy_coordinate=False):
     """Calculate the ious between each bbox of bboxes1 and bboxes2.
 
     Args:
-        bboxes1(ndarray): shape (n, 4)
-        bboxes2(ndarray): shape (k, 4)
-        mode(str): iou (intersection over union) or iof (intersection
+        bboxes1 (ndarray): Shape (n, 4)
+        bboxes2 (ndarray): Shape (k, 4)
+        mode (str): IOU (intersection over union) or IOF (intersection
             over foreground)
+        use_legacy_coordinate (bool): Whether to use coordinate system in
+            mmdet v1.x. which means width, height should be
+            calculated as 'x2 - x1 + 1` and 'y2 - y1 + 1' respectively.
+            Default: False.
 
     Returns:
-        ious(ndarray): shape (n, k)
+        ious (ndarray): Shape (n, k)
     """
 
     assert mode in ['iou', 'iof']
-
+    if not use_legacy_coordinate:
```

Comment:
Why not add the extra_length to bbox before passing them to the bbox_overlaps function?

Reply:
I believe that all metric-related code should strictly follow the official implementation, and equivalent modification may cause unnecessary misunderstandings.